### PR TITLE
Updates the latest releases in libs.md

### PR DESCRIPTION
**Pull request by zslayton**
Updates:
* `ion-c` from 1.0.3 to 1.4.0
* `ion-go` from 1.0.1 to 1.1.0
* `ion-java` from 1.8.0 to 1.8.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by desaikd — APPROVED** [review]
LGTM!

File: libs.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ description: "The latest news about Amazon Ion and the Amazon Ion community."
...
 |------|----------------|------|---------------|
-| ion-c | [1.0.3](https://github.com/amzn/ion-c/releases/latest) (December 11, 2019) | [Link](https://github.com/amzn/ion-c) | [Link](https://amzn.github.io/ion-c/) |
+| ion-c | [1.4.0](https://github.com/amzn/ion-c/releases/latest) (February 9, 2021) | [Link](https://github.com/amzn/ion-c) | [Link](https://amzn.github.io/ion-c/) |
 | ion-dotnet | [1.1.0](https://github.com/amzn/ion-dotnet/releases/latest) (November 24, 2020) | [Link](https://github.com/amzn/ion-dotnet) | - |
-| ion-go | [1.0.1](https://giithub.com/amzn/ion-go/releases/latest) (October 16, 2020) | [Link](https://github.com/amzn/ion-go) | - |
-| ion-java | [1.8.0](https://github.com/amzn/ion-java/releases/latest) (November 9, 2020) | [Link](https://github.com/amzn/ion-java) | [Link](https://www.javadoc.io/doc/com.amazon.ion/ion-java/) |
+| ion-go | [1.1.0](https://github.com/amzn/ion-go/releases/latest) (December 8, 2020) | [Link](https://github.com/amzn/ion-go) | - |
+| ion-java | [1.8.1](https://github.com/amzn/ion-java/releases/latest) (April 7th, 2021) | [Link](https://github.com/amzn/ion-java) | [Link](https://www.javadoc.io/doc/com.amazon.ion/ion-java/) |
```
Comment (jonwilsdon) [inline libs.md]: `April 7th` to `April 7` for consistency.

**Review by jonwilsdon — APPROVED** [review]
One minor change, otherwise it looks good.

File: libs.md (hunk 1 above)
Comment (zslayton) [reply libs.md]: Oops. 🤦 Good catch!

**Pull request merged**